cmd/geth/consolecmd.go

```diff
@@ -134,6 +134,9 @@ func remoteConsole(ctx *cli.Context) error {
 				path = filepath.Join(path, "rinkeby")
 			} else if ctx.GlobalBool(utils.GoerliFlag.Name) {
 				path = filepath.Join(path, "goerli")
+			} else if ctx.GlobalBool(utils.MumbaiFlag.Name) || ctx.GlobalBool(utils.BorMainnetFlag.Name) {
```

Comment:
How about using different paths for Bor Mainnet vs Mumbai? Just like Go-ethereum does?

Reply:
I agree, But currently, in our default node setup, we use the same path for bor irrespective of the network, So to avoid that confusion I did the same.